### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by NabilHY**
In this branch I have :
>Removed all hardcoded items from the tasks array.
> Implemented functions for adding, removing, and editing tasks.
> Made Sure that both the task indexes description and status gets updated both on the UI and the LS

### CODE REVIEW

File: check.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  const buttons = boxEle.parentNode.lastElementChild;
+  const task = (boxEle.nextElementSibling);
+  buttons.classList.remove('none');
+  task.classList.remove('completed');
+};
+
+const editVal = (index) => {
+  const arr = JSON.parse(localStorage.getItem('tasks'));
+  arr.forEach((task) => {
+    // eslint-disable-next-line eqeqeq
```
Comment (bimbolabuari) [inline check.js]: [Optional] Kindly remove all disabled Linters, as it helps in writing more professional code

File: editFunc.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable eqeqeq */
```
Comment (bimbolabuari) [inline editFunc.js]: [Optional] Kindly remove all disabled Linters, as it helps in writing more professional code

Please check for other occurrences 👍

**Review by bimbolabuari — APPROVED** [review]
Hi @NabilHY, 

Your project is complete!  It's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: check.js (hunk 1 above)
Comment (NabilHY) [reply check.js]: I appreciate your feedback, the problem was addressed and solved.
Thank you.

**Pull request merged**